core/common/src/main/java/alluxio/client/file/CacheContext.java

```diff
@@ -152,6 +152,18 @@ public CacheContext setTemporary(boolean isTemporary) {
     return this;
   }
 
+  /**
+   * Increments the counter {@code name} by {@code value}.
+   * <p>
+   * Default implementation does nothing. Subclass can implement its own tracking mechanism.
+   *
+   * @param name name of the counter
+   * @param value value of the counter
+   */
+  public void incrementCounter(String name, long value) {
+    // Default implementation does nothing
```

Comment:
if we depends on the old method, do we still want the old implementation by calling the three parameter new method here?